src/App.js

```diff
@@ -1,8 +1,10 @@
+import './App.css';
+import Logic from './components/Logic.js';
 
 function App() {
   return (
     <div className="App">
-
+      <Logic />
     </div>
   );
 }
```

Comment:
- [ ] This component should be in the components folder, as mentioned in the activity. Kindly move it
- [ ] The component should render two other components `Display` and `ButtonPanel`. Kindly implement as explained in the activity